ci/docker/qemu/qemu_run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -exuo pipefail
+
+qemu-system-arm -M virt -m 1024 \
+  -kernel vmlinuz-3.16.0-6-armmp-lpae \
```

Comment:
I think if we could alias or rename the versioned image the whole pipeline would be much more stable

Reply:
what do you mean? can you explain?

Reply:
You download a file `vmlinuz-3.16.0-6-armmp-lpae`, but rename it to `kernel`. Then in all other places this name is used and in the case of update you don't need to change it.